src/routes/process-user-vocabulary/UserVocabularyProcessScreen.tsx

```diff
@@ -52,10 +56,53 @@ const ThumbnailContainer = styled.View`
   flex-direction: row;
 `
 
+const AudioContainer = styled(Animated.View)`
+  margin: ${props => props.theme.spacings.md} 0;
+  justify-content: flex-start;
+  padding: 0;
+  flex-direction: row;
+`
+
+const DeleteContainer = styled.Pressable`
+  align-self: center;
+  justify-content: center;
+  align-items: center;
+  margin: ${props => `${props.theme.spacings.xs} ${props.theme.spacings.md}`};
+  shadow-color: ${props => props.theme.colors.shadow};
+  elevation: 8;
+  shadow-radius: 5px;
+  shadow-offset: 1px 1px;
+  shadow-opacity: 0.5;
+`
+
+const PlayContainer = styled.Pressable<{ isActive: boolean }>`
+  align-self: center;
+  justify-content: center;
+  align-items: center;
+  background-color: ${props =>
+    props.isActive ? props.theme.colors.audioIconSelected : props.theme.colors.audioIconHighlight};
+  border-radius: 50px;
+  shadow-color: ${props => props.theme.colors.shadow};
+  elevation: 8;
+  shadow-radius: 5px;
+  shadow-offset: 1px 1px;
+  shadow-opacity: 0.5;
+`
+
+const AudioText = styled(Subheading)`
+  align-self: center;
+`
+
 interface UserVocabularyProcessScreenProps {
   navigation: StackNavigationProp<RoutesParams, 'UserVocabularyProcess'>
 }
 
+const accuracy = 0.1
+const playTolerance = 200
+
+const audioRecorderPlayer = new AudioRecorderPlayer()
+audioRecorderPlayer.setSubscriptionDuration(accuracy).catch(e => e)
+
 const UserVocabularyProcessScreen = ({ navigation }: UserVocabularyProcessScreenProps): ReactElement => {
```

Comment:
i used our `AudioRecorder` component. since we use the `fadeIn` after setRecording path its a little bit complicated to extract the Animation. If you want you can create a separate component for the animation and refactor it

Reply:
put everything in a new component